src/libs/SidebarUtils.ts

```diff
@@ -179,15 +178,13 @@ function getOrderedReportIDs(
     //      - Sorted by lastVisibleActionCreated in default (most recent) view mode
     //      - Sorted by reportDisplayName in GSD (focus) view mode
     const pinnedReports: Report[] = [];
-    const outstandingIOUReports: Report[] = [];
     const draftReports: Report[] = [];
     const nonArchivedReports: Report[] = [];
     const archivedReports: Report[] = [];
     reportsToDisplay.forEach((report) => {
-        if (report.isPinned) {
+        const isPinned = report.isPinned ?? false;
+        if (isPinned || ReportUtils.shouldShowGBR(report)) {
```

Comment:
Is the desired behavior that pinned reports and GBR reports are "interspersed" with each other? Or should GBR be its own distinct group?

Reply:
Yep, desired that they're treated the same. Should I leave a comment explaining that?

Reply:
Or actually, I'll change the variable name.

Reply:
👍

Reply:
kk updated!